src/gConsent/manage/approveAccessRequest.test.ts

```diff
@@ -311,6 +312,24 @@ describe("approveAccessRequest", () => {
     );
   });
 
+  it("accepts the returned VC using the abbreviated status instead of the fully qualified IRI", async () => {
+    mockAccessApiEndpoint();
+    const mockedVcModule = jest.requireMock(
+      "@inrupt/solid-client-vc"
+    ) as typeof VcClient;
+    const mockedIssue = jest.spyOn(mockedVcModule, "issueVerifiableCredential");
+    const mockedVc = mockAccessGrantVc();
+    mockedVc.credentialSubject.providedConsent.hasStatus =
```

Comment:
Can we also update other places these IRIs are used with the constants? e..g, line 299 in this file.

Reply:
Actually, I will do it the other way around and update this to use the string value: 941bbe788c1dcea355d37a530f3e1ba1ee6a4f24. The reason is that if we use the constants in the tests and in the code, we could update the constants to whatever value and the tests would still pass, whereas having the actual value in the test shows intent of what the constant should be.

Reply:
Good call.